### PR TITLE
Update `README.md` with installation guide section

### PR DESCRIPTION
**Pull request by Shadowman405**
## Merge after

- #39 

## List of Changes

- Updated `README.md` with installation guide section.

### CODE REVIEW

**Review by auhustsinovich — CHANGES_REQUESTED** [review]
Please consider minor improvements:

**Review by auhustsinovich — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -23,11 +23,30 @@ This repository contains:
...
-1. Download the latest release version of the `LifoCollections.xcramework` from this repository.
-2. Add the `LifoCollections.xcramework` to your project. We recommend you to choose the `Do Not Embed` option.
+### Swift Package Manager
+
+The `LifoCollections` framework supports installation via [Swift Package Manager](https://swift.org/package-manager/) (SPM) from version `0.2.0`.
+
+You can add `LifoCollections` by adding it as a package dependency to your Xcode project:
+1. In Xcode project, select 'File' -> 'Add Package Dependencies...'.
+1. Enter `https://github.com/Shadowman405/LifoCollections.git` into 'Search or Enter Package URL' search bar.
+1. Select the version of the `LifoCollections` package you want to use. For new projects, recommend using the 'Up to Next Major Version' dependency rule.
```
[the remark this answers is not in the document]
Comment (Shadowman405) [reply README.md:35]: [Here](https://github.com/Shadowman405/LifoCollections/pull/42#discussion_r1617397063) I did opposite fixes. Maybe I should leave this line as it is ?
It seemed to say that it is better not to use the word `we` in the documentation.

File: README.md [hunk 2]
```diff
@@ -23,11 +23,30 @@ This repository contains:
...
+1. Click 'Add Package' button.
+
+### XCFramework
+
+Each release includes `*.xcframework` framework binary.
+
+1. Download `LifoCollections.xcframework.zip` from [releases](https://github.com/Shadowman405/LifoCollections/releases).
+1. Open the Xcode project or workspace to which you want to add the `LifoCollections.xcframework`.
+1. Go to the target’s General configuration page.
+1. Add the framework target to the Frameworks, Libraries, and Embedded Content section by clicking the `Add(+)` button.
```
[the remark this answers is not in the document]
Comment (Shadowman405) [reply README.md]: Fixed ✅

**Review by AlexLevshenkov — APPROVED** [review]

**Pull request merged**